test/import.bats

```diff
@@ -72,3 +72,36 @@ EOF
 
     stacker build
 }
+
+@test "different import types" {
+    touch test_file
+    test_file_sha=$(sha test_file) || { stderr "failed sha $test_file"; return 1; }
+    touch test_file2
+    cat > stacker.yaml <<EOF
+first:
+    from:
+        type: oci
+        url: $CENTOS_OCI
+    import:
+        - path: test_file
+          hash: $test_file_sha
```

Comment:
Can we test the failure case too, where the sha doesn't match? There should be a `stacker_bad` function you can use to expect failure of stacker.

Reply:
Ok, sure, thank you.